src/components/EmojiPicker/EmojiPickerMenu/BaseEmojiPickerMenu.tsx

```diff
@@ -126,6 +126,8 @@ function BaseEmojiPickerMenu(
                             scrollPaddingTop: isFiltered ? 0 : CONST.EMOJI_PICKER_ITEM_HEIGHT,
                         },
                     }}
+                    // Fix for choppy scrolling on iOS Safari, https://github.com/Expensify/App/issues/57183
```

Comment:
```suggestion
```

Not a blocker, but this is redundant as the same info can be obtained from git blame